tests/test_selection.py

```diff
@@ -283,6 +284,14 @@ def test_top(ala):
     eq(np.asarray(wat), np.arange(22, 2269))
 
 
+def test_atom_slice(get_fn):
```

Comment:
This test should likely be moved somewhere more 'pdb' centric as it is really a slice+pdb write that is the issue being addressed.